coderd/database/migrations/000126_login_type_none.up.sql

```diff
@@ -1,3 +1,34 @@
-ALTER TYPE login_type ADD VALUE IF NOT EXISTS 'none';
+-- This migration has been modified after its initial commit.
+-- The original disregarded the warning in create_migration.sh.
+-- As a result, it was not possible to insert a user with the "none" login type
+-- in a migration.
+-- The new implementation makes the same changes as the original, but
```

Comment:
Nit: this comment should rather just going into the PR description, and the original argument should be copied over as well. Linking to a GH commit might not always work, or be difficult to find once the comment is resolved.

Reply:
Sounds good; for consistency should it not go in both files then?